### PR TITLE
Improve README documentation

### PR DESCRIPTION
**Pull request by dlebech**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -77,11 +75,70 @@ app := goshopify.App{
...
+type WebhooksResource struct {
+    Webhooks []Webhook `json:"webhooks"`
+}
+
+func FetchWebhooks() ([]Webhook, error) {
+	path := "admin/webhooks.json"
+	resource := new(WebhooksResoure)
+    client := goshopify.NewClient(app, "shopname", "token")
+
+    // resource gets modified when calling Get
```
Comment (jmnsf) [inline README.md]: glitchy indent on this and above line

Comment (jmnsf) [inline README.md]: or maybe these 2 are the only ones correct

**Comment by jmnsf** [comment]
:+1: indentation issue

File: README.md (hunk 1 above)
Comment (stefanosala) [inline README.md]: oh tabs, why tabs? :)

**Comment by stefanosala** [comment]
👍

File: README.md (hunk 1 above)
Comment (dlebech) [inline README.md]: Some of it is copy-pasted from the actual code which explains the tabs. I'll convert it to spaces for the README :)

**Pull request merged**